src/components/Dashboard/DashboardData.js

```diff
@@ -31,11 +32,11 @@ function DashboardData() {
     }
   }, [user, userSitesIsLoading, userSitesError])
 
-  const { roles, isSuperAdmin } = user
+  const { roles } = user
 
   const currentUser = {
     ...user,
-    isAdmin: !!(roles?.[sitename] === 'Admin' || isSuperAdmin),
+    isAdmin: user?.isLanguageAdmin,
```

Comment:
Just realized this is wrong-- isLanguageAdmin is not site-specific. I'll change it back.